com.unity.cinemachine/Documentation~/CinemachineInputAxisController.md

```diff
@@ -28,3 +31,43 @@ If you like, you can also use CinemachineInputAxisController with your own scrip
 | __Accel Time__ | The time it takes for the input value to accelerate to a larger value |
 | __Decel Time__ | The time it takes for the input value to decelerate to a smaller value |
 
+## Creating your own Input Axis Controller
+
+The default implementation of `CinemachineInputAxisController` can process input from the Input package and from Unity's legacy input system. In the case where you want your input to come from an other source, you may need to create your own input controller. This example shows how to control a Camera with a slider. It can be easily modified to be used with other objects.
+
+```cs
+using Unity.Cinemachine;
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+[Serializable]
+public class SliderReader : IInputAxisReader 
```

Comment:
It's a convention with interfaces. You must add an I before the name of the class. It can lead to weird cases like that but it's normal and expected to be like that.